region/brinstar/blue/Morph Ball Room.json

```diff
@@ -241,6 +241,51 @@
         "Using the full runway, this requires a last-frame jump, followed by a 2-frame window for the morph."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_ZebesIsAwake",
+        {"or": [
+          "ScrewAttack",
+          {"and": [
+            {"enemyDamage": {"enemy": "Sidehopper", "type": "contact", "hits": 1}},
+            {"enemyKill": {
+              "enemies": [["Sidehopper", "Sidehopper"]],
+              "explicitWeapons": ["Missile", "Super", "PowerBomb", "ScrewAttack", "Plasma"]
```

Comment:
- ScrewAttack can be removed
- Powerbombs can work but I don't know what the best course is.  Using it on the left can kill the ceiling hopper & running to the right might mean opening the PB wall for runway which is another requirement & slow killing the ground hoppers makes it hard to collect their drop with the ceiling hopper alive.